modules/order/commerce_order.module

```diff
@@ -26,6 +26,10 @@ function commerce_order_theme($existing, $type, $theme, $path) {
     'commerce_order_edit_form' => [
       'render element' => 'form',
     ],
+    'commerce_order__default' => [
```

Comment:
Discussed w/ Bojan. Doesn't really give us a gain. And tests showed all kinds of madness by doing this.